src/main/java/uk/gov/companieshouse/model/Attachment.java

```diff
@@ -0,0 +1,78 @@
+package uk.gov.companieshouse.model;
+
+import java.util.Objects;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+
+public class Attachment {
+
+    @NotBlank(message = "attachement id must not be null.")
+    private String id;
+
+    @NotBlank(message = "name must not be blank.")
+    private String name;
+
+    @NotBlank(message = "contentType must not be blank.")
+    private String contentType;
```

Comment:
we could add some additional validation for the content type as we do on the front end?

Reply:
I think for now we can get away with trusting the front end